src/apps/material/material.tsx

```diff
@@ -125,7 +125,10 @@ const Material: React.FC<MaterialProps> = ({ wid }) => {
                 manifestation={manifestation}
               />
               <FindOnShelfModal
-                manifestation={manifestation}
+                manifestations={[manifestation]}
+                workTitles={work.titles.full}
+                authors={work.creators}
+                pid={manifestation.pid}
                 key={`find-on-shelf-modal-${manifestation.pid}`}
```

Comment:
You can destruct all the work properties in the definition of the `Material` functional component

Reply:
I'm sorry but I think I prefer it not destructed 🙈